### ISSUE TITLE
Polish Infographic for Social Media

### ISSUE THREAD

**Issue by notconfusing** [notconfusing/WIGI#22]
Edit the WVC edition to including at least
- a graphic in the top right
- a final blurb "We still don't know exactly how the bias of Wikipedia's editorship and the biases in the real-world interact. Studying Wikidata over time in combination with different gender-focused intiatives, we hope that the WIGI project can tell us more."

**Comment by fsoong** [comment]
Updated. Please review for any discrepancies and feedback.
Please keep in mind that what you see here in Github is not the full-size. The actual infographic is larger.
![wigi_infographic_v05_20150718](https://cloud.githubusercontent.com/assets/12237852/8762332/28eb4102-2d2b-11e5-8759-529a1e27bc7e.png)

UPDATED July 17, 2015: Missing line above 40% is added back in.
UPDATED July 18, 2015: "English" reverted to "English Speaking".

**Comment by notconfusing** [comment]
One more polishing thing to do, was to revert the culture label from "English" back to "English Speaking". In fact, it would be great if you could post the .AI file here too.

**Comment by fsoong** [comment]
I can't post anything other than PNG, GIF, or JPG onto Github so I'll be emailing them to you. 
Here is a copy of the CSV data I used and previous version so everyone can compare. There is a lot of information in this one, and I can't promise I got every detail and fact straight. I'd really appreciate it if someone had time to comb through it and look for any discrepancies.

https://www.evernote.com/l/ANseF4J8CslLBbD_Trj0NURSkDvqqsyO0lc
